types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
"The first sentence should be a one-sentence summary that starts with the name being declared."

This should be:

```
// LiveRestore determines ...
```